src/NuGet.Clients/NuGet.Tools/NuGetPackage.cs

```diff
@@ -1213,6 +1231,93 @@ private void ShowGeneralSettingsOptionPage(object sender, EventArgs args)
             ShowOptionPageSafe(typeof(GeneralOptionPage));
         }
 
+        private void ExecuteClearNuGetLocalResourcesCommand(object sender, EventArgs e)
```

Comment:
This class is already very long, can we move this method into a different class?

If you create a new VSIX project from VS's template, and use the wizard to add a new command, you'll see it creates the command in a separate file/class, which I think is more maintainable. I once looked into into moving some of the "commands" already in this file into separate files, but it was a bit of a spaghetti mess with inter-dependencies, so I gave up.   I'm hoping we can avoid increasing the tech debt on new commands.

Reply:
It is quite long. Good point. Refactoring is done!